epfast-annotations/src/main/java/com/exactpro/epfast/converter/Converter.java

```
@@ -0,0 +1,166 @@
+    }
+
+    public BigInteger integerToBigInteger(Integer s) {
+        return BigInteger.valueOf(s);
+    }
+
+    public Long integerToLong(Integer s) {
+        return Long.valueOf(s);
+    }
+
+    public String integerToString(Integer s) {
+        return String.valueOf(s);
+    }
+
+    // Long conversions
+    public BigDecimal longToDecimal(Long s) {
+        return BigDecimal.valueOf(s);
+    }
+
+    public BigInteger longToBigInteger(Long s) {
+        return BigInteger.valueOf(s);
+    }
+
+    public Integer longToInteger(Long s) throws Exception {
+        if (s > Integer.MAX_VALUE || s < Integer.MIN_VALUE) {
+            throw new Exception(String.format("%s cannot be converted to Integer", s));
+        }
+        return Integer.valueOf(String.valueOf(s));
+    }
+
+    public String longToString(Long s) {
+        return String.valueOf(s);
+    }
+
+    // BigInteger conversions
+    public BigDecimal bigIntegerToDecimal(BigInteger s) {
+        return new BigDecimal(s);
+    }
+
+    public Long bigIntegerToLong(BigInteger s) throws Exception {
+        long value = s.longValue();
+        if (!BigInteger.valueOf(value).equals(s)) {
+            throw new Exception(String.format("%s cannot be converted to Long", s));
+        }
+        return value;
+    }
+
+    public Integer bigIntegerToInteger(BigInteger s) throws Exception {
+        int value = s.intValue();
+        if (!BigInteger.valueOf(value).equals(s)) {
+            throw new Exception(String.format("%s cannot be converted to Integer", s));
+        }
+        return value;
+    }
+
+    public String bigIntegerToString(BigInteger s) {
+        return String.valueOf(s);
+    }
+
+    // String conversions
+    public BigDecimal stringToDecimal(String s) throws Exception {
+        try {
+            return new BigDecimal(s);
+        } catch (NumberFormatException e) {
+            throw new Exception(String.format("'%s' cannot be converted to BigDecimal", s));
```

`NumberFormatException` is good enough. just let it be thrown